### PR TITLE
Add parallax recipe

### PR DESCRIPTION
**Pull request by aleku399**
parallax

### CODE REVIEW

File: ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -442,7 +445,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
-				DEVELOPMENT_TEAM = M7AKZYT269;
+				DEVELOPMENT_TEAM = YHM24P2423;
```
Comment (epicallan) [inline ios/Runner.xcodeproj/project.pbxproj:448]: Suggestion: If I am not mistaken, the changes in this file seem unrelated to the core task as per the PR title. They should be in a separate commit.

**Review by epicallan — APPROVED** [review]
Everything seems fine to me but my only worry is that you don't have any tests or any CI linting tooling.

File: ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (aleku399) [reply ios/Runner.xcodeproj/project.pbxproj:448]: yes

**Comment by aleku399** [comment]
> Everything seems fine to me but my only worry is that you don't have any tests or any CI linting tooling.

will focus on them seperately

**Comment by epicallan** [comment]
I think you should merge this PR if you are happy with it and move on to the next task.
